### DISCUSSION TITLE
Tactile Feedback on Zwift Play controllers

### DISCUSSION THREAD

**Discussion by GraVvVel** [cagnulein/qdomyos-zwift#2750] [Ideas]
When using the Zwift Play controllers in Zwift directly they provide subtle tactile feedback for each gear shift, which is a nice touch and when shifting multiple gears also provides additional feedback. I would be great to add this in QZ too.

**Comment by cagnulein** [comment]
thanks @GraVvVel you can use the issues instead of the discussion the next time, thanks!

**Reply by GraVvVel** [reply]
Ok - should I also add the multi-shift as an issue (I thought it's like an additonal feature/idea - but you tell me how to prefer it)?

**Reply by cagnulein** [reply]
yes please, issue then feature request! thanks
